src/common/language_eeprom.cpp

```diff
@@ -0,0 +1,46 @@
+
+#include "language_eeprom.hpp"
+#include "../lang/translator.hpp"
+
+/*!
+ * Setter & getter class to store language code into a EEPROM
+ * Constructor will check if language code is not set, then it will set
+ * a default value.
+ * If it's set, then Translator is called to start translating.
+ */
+LangEEPROM::LangEEPROM()
+    : _language(0) {
+    uint16_t _language = static_cast<uint16_t>(eeprom_get_var(EEVAR_LANGUAGE).ui16);
+    if (_language == static_cast<uint16_t>(0xffff)) {
```

Comment:
default lang should be "en"

Reply:
sure, that was just a test - it's gonna be changed